source/databricks/tests/integration/calculator/balance_fixing_total_production/test_get_enriched_time_series_points_df.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pytest
+from decimal import Decimal
+from datetime import datetime
+from package.balance_fixing_total_production import _get_enriched_time_series_points_df
+from tests.contract_utils import (
+    assert_contract_matches_schema,
+    read_contract,
+    get_message_type,
+)
+
+
+@pytest.fixture(scope="module")
+def raw_time_series_points_factory(spark, timestamp_factory):
+    def factory(
+        time: datetime = timestamp_factory("2022-06-08T12:09:15.000Z"),
+    ):
+        df = [
+            {
+                "GsrnNumber": "1",
+                "TransactionId": "1",
+                "Quantity": Decimal("1.1"),
+                "Quality": 3,
+                "Resolution": 2,
+                "RegistrationDateTime": timestamp_factory("2022-06-10T12:09:15.000Z"),
+                "storedTime": timestamp_factory("2022-06-10T12:09:15.000Z"),
+                "time": time,
+                "year": 2022,
+                "month": 6,
+                "day": 8,
+            }
+        ]
+        return spark.createDataFrame(df)
+
+    return factory
+
+
+@pytest.fixture(scope="module")
+def metering_point_period_df_factory(spark, timestamp_factory):
+    def factory(
+        effective_date: datetime = timestamp_factory("2022-06-08T12:09:15.000Z"),
+        to_effective_date: datetime = timestamp_factory("2022-06-08T13:09:15.000Z"),
+    ):
+        df = [
+            {
+                "MessageType": "1",
+                "GsrnNumber": "1",
+                "GridAreaCode": "1",
+                "EffectiveDate": effective_date,
+                "toEffectiveDate": to_effective_date,
+                "Resolution": 2,
```

Comment:
Perhaps something more like:
```suggestion
                "GsrnNumber": "the-gsrn-number",
                "GridAreaCode": "805",
                "EffectiveDate": effective_date,
                "toEffectiveDate": to_effective_date,
                "Resolution": TimeSeriesResolution.hour,
```

Reply:
I won't import Resolution because its not being used in the test